javagen/src/main/java/com/azure/autorest/mapper/ProxyMethodMapper.java

```diff
@@ -23,14 +25,14 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-public class ProxyMethodMapper implements IMapper<Operation, ProxyMethod> {
+public class ProxyMethodMapper implements IMapper<Operation, Map<Request, ProxyMethod>> {
     private static final List<IType> unixTimeTypes = Arrays.asList(PrimitiveType.UnixTimeLong, ClassType.UnixTimeLong, ClassType.UnixTimeDateTime);
     private static final List<IType> returnValueWireTypeOptions = Stream.concat(Stream.of(ClassType.Base64Url, ClassType.DateTimeRfc1123), unixTimeTypes.stream()).collect(Collectors.toList());
     private static ProxyMethodMapper instance = new ProxyMethodMapper();
 
 //    private static final jdk.nashorn.internal.runtime.regexp.joni.Regex methodTypeLeading = new Regex("^/+");
 //    private static final Regex methodTypeTrailing = new Regex("/+$");
-    private Map<Operation, ProxyMethod> parsed = new HashMap<Operation, ProxyMethod>();
+    private Map<Request, ProxyMethod> parsed = new HashMap<>();
```

Comment:
An operation can now generate multiple proxy methods, per request.

Reply:
I see. The input of json schema and input of binary upload is put to 2 different requests...